### PR TITLE
Update ebmac from 1.45.0 to 1.45.1

### PR DESCRIPTION
**Pull request by suschizu**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/ebmac.rb [hunk 1]
```diff
@@ -1,8 +1,9 @@
 cask 'ebmac' do
-  version '1.45.0'
-  sha256 '969248c8c9caec20d131a7f2da263f4a2acc5e4ac510de9bcb3c0a664711f8ff'
+  version '1.45.1'
+  sha256 '91f454e709607ae8b397b13e4201adc4add8e80394969918218fc097831f3c73'
 
-  url "http://ebstudio.info/download/ebpocket/EBMac#{version}.dmg"
+  # ftp.vector.co.jp/72/54/938 was verified as official when first introduced to the cask
+  url "http://ftp.vector.co.jp/72/54/938/EBMac#{version}.dmg"
```
Comment (vitorgalvao) [inline Casks/ebmac.rb]: How do we get here from the home?

Comment (core-code) [reply Casks/ebmac.rb]: http://ebstudio.info
=>
http://ebstudio.info/manual/EBMac/
=>
https://www.vector.co.jp/soft/mac/writing/se488130.html

Comment (suschizu) [reply Casks/ebmac.rb]: they moved the download to this strange site - there is no other download on the website unfortunately

Comment (vitorgalvao) [reply Casks/ebmac.rb]: It’s beyond me why some vendors are so bad and giving people a download for their apps.

Comment (core-code) [reply Casks/ebmac.rb]: especially when web-hosting is unbelievably cheap these days...

**Pull request merged**